OCAPP/Models/Conference.py

```diff
@@ -41,8 +42,17 @@ def register(id, member,data):
 	mc.conference = conf
 	mc.member = member
 	mc.days = data['days']
-	print mc
 	db.session.add(mc)
 	db.session.commit()
 	return conf
 
+def set_transaction(conf_id, member_id, transaction_id):
+	conf = db.get(Conference, conf_id)
+	memb = Member.get_by_id(member_id)
+	for member_conf in conf.members:
+		if member_conf.member_id == memb.id:
+			member_conf.transaction_id = transaction_id
+			member_conf.member_paid = True
+			db.session.add(member_conf)
+			db.session.commit()
+	return 
```

Comment:
This should return something, even if not utilizing it right now.  Why not return the member_paid status?